_haikus/koala.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Koala
+author: urbanfog
+---
+
+Two thumbs on each hand  
+stuffing, chewing, eating leaves.  
+Too tired, must have more.
```

Comment:
One syllable too many here! We have 6 currently, but should be 5. A suggestion to modify this:
```
Tired, must have more.
```

Reply:
"tired" is another one of those ambiguously-syllabled words. mostly i see it listed as 1 syllable. since the CI robot agrees – and in the interest of keeping PRs moving here – i will resolve and merge.